INSTALL

```diff
@@ -4,10 +4,7 @@
 
 extsmail follows the standard UNIX installation sequence:
 
+  $ autoreconf -i
```

Comment:
People downloading a release don't need to do this, right? They just need to do `./configure`? If so, maybe we can keep the "please note" paragraph, changing `make -f Makefile.boostrap` to `autoreconf -i`?